docs/build/html/_sources/app.rst

```diff
@@ -0,0 +1,109 @@
+App Module
+==========
+
+The `App` Module contains two files in it, `image_processor.py` and `input_manager.py`. When the application starts,
+it's these two files that initiate the process of processing the given images and accepting user input respectively.
+
+
+Input Manager
+-------------
+
+``start()``
+This method will start the application and is called in the `runner.py` file. It will first prompt the user for a
```

Comment:
This is a common issue in this file try not to start a documentation with "This method"